examples/mobile/HomeApp/js/app.js

```diff
@@ -77,6 +81,7 @@
 	function onPopupSubmit() {
 		getWidgetsState();
 		toggleWidgetsState();
+		storage.readFromUI(event.target)
```

Comment:
readFromUI returns value. Is it used anywhere?

Reply:
Done!